common/src/main/java/com/wynntils/services/usermarker/type/UserMarkerLocation.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright © Wynntils 2025.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.services.usermarker.type;
+
+import com.wynntils.core.components.Services;
+import com.wynntils.services.mapdata.attributes.MapMarkerOptionsBuilder;
+import com.wynntils.services.mapdata.attributes.impl.MapLocationAttributesImpl;
+import com.wynntils.services.mapdata.attributes.type.MapLocationAttributes;
+import com.wynntils.services.mapdata.features.impl.MapLocationImpl;
+import com.wynntils.services.mapdata.providers.builtin.MapIconsProvider;
+import com.wynntils.utils.colors.CommonColors;
+import com.wynntils.utils.mc.type.Location;
+import com.wynntils.utils.render.Texture;
+import java.util.Optional;
+
+public final class UserMarkerLocation extends MapLocationImpl {
+    private final String name;
+    private final MapLocationAttributesImpl userMarkerAttributes;
+
+    public UserMarkerLocation(Location location, String name) {
+        super("user-marker-" + location.hashCode(), "wynntils:personal:user-marker", null, location);
+        this.name = name;
+        this.userMarkerAttributes = Services.UserMarker.MARKED_MAP_FEATURE_ATTRIBUTES
```

Comment:
This is kind of an anti-pattern for MapData. The location itself should only provide the attributes that are changing and specific to this particular Location. In this particular case, it is just the `name`. The rest of these attributes are default values which should be set on the the category instead, so that it can be easily overridden for all locations of this kind.

We might have to check all attribute creations if we are doing this.

Reply:
I think this is the only place. This should be an easy fix, thankfully.

Reply:
Let's hope it is. But I think it would be good to remember to double-check this before merging the final mapdata-port branch.